web/functions/index.js

```diff
@@ -1,18 +1,26 @@
 // This is where our endpoints go
 const functions = require("firebase-functions");
-const admin = require("firebase-admin"); // the admin sdk
 const express = require("express");
+const {db} = require('./util/admin')
+const { signUp, login} = require('./routes/user')
+const {uploadImage} = require('./routes/imgUpload')
+const auth = require('./middleware/auth')
 const app = express();
```

Comment:
we can delete this right #
```suggestion
const app = express();
```

Reply:
No we can't. Nothing will work if we delete that line.